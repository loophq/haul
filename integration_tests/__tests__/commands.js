/**
 * Copyright 2017-present, Callstack.
 * All rights reserved.
 *
 * @flow
 */

const runHaul = require('../runHaul');
const { cleanup, writeFiles } = require('../utils');
const path = require('path');
const os = require('os');

const DIR = path.resolve(os.tmpdir(), 'commands_test');

beforeEach(() => cleanup(DIR));
afterEach(() => cleanup(DIR));

test('start command displays "Select platform" message', () => {
  writeFiles(DIR, {
    'webpack.haul.js': '{}',
  });

  const { stdout } = runHaul(DIR, ['start', '--platform ios']);
  expect(stdout).toMatchSnapshot();
});
