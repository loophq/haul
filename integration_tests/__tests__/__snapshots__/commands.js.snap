// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`start command displays "Select platform" message 1`] = `
"? Select platform to bundle for (Use arrow keys)
‚ùØ ios - Serves iOS bundle 
  android - Serves Android bundle 
  all - Serves both platforms [30D[30C
"
`;
